### PR TITLE
[docs]: Fix documentation for Statistics Component

### PR DESCRIPTION
**Pull request by baibhavKumar1**
This PR addresses the following changes in the documentation for the Statistics component:

Formatting updates:

Replaced "Fx" with "fx" wherever applicable.
Removed 24px padding-bottom for all h2 headers, ensuring only 24px padding-top remains in the wrapping div.

Content updates:

Enhanced the introduction with a more descriptive summary.
Replaced the term "widget" with "component" throughout the documentation (excluding URLs).
Added a divider before the "Styles" section for better content structure.

### CODE REVIEW

**Comment by baibhavKumar1** [comment]
#10901 completed

File: docs/docs/widgets/statistics.md [hunk 1]
```diff
@@ -5,63 +5,65 @@ title: Statistics
...
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 | <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> |
 |:------------ |:-------------|
 | Primary value label | This property lets you add and remove primary value label. | 
 | Primary value | This property lets you add and remove primary value, the actual amount can be represented here. |
-| Hide secondary value | This property lets you show/hide the secondary value from the statistics component. By default, this property is disabled. Toggle it on to hide the secondary value, you can also click on the `Fx` button next to it to dynamically set the value to `{{true}}` or `{{false}}`. |
+| Hide secondary value | This property lets you show/hide the secondary value from the statistics component. By default, this property is disabled. Toggle it on to hide the secondary value, you can also click on the `fx` button next to it to dynamically set the value to `{{true}}` or `{{false}}`. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/statistics.md:18]: Hey @baibhavKumar1 the fx should be bold and not enclosed in backticks, can you please make this change in the whole doc and both versions wherever applicable!

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @baibhavKumar1 please make the mentioned change in both the version.

File: docs/docs/widgets/statistics.md (hunk 1 above)
Comment (baibhavKumar1) [reply docs/docs/widgets/statistics.md:18]: @RudraPratik30 just in statistics.md or whole doc?

Comment (baibhavKumar1) [reply docs/docs/widgets/statistics.md]: Resolved in statistics.md in both version.

File: docs/docs/widgets/statistics.md [hunk 2]
```diff
@@ -5,63 +5,65 @@ title: Statistics
 
 # Statistics
 
-**Statistics** can be used to display different statistical information.
+**Statistics** help us understand and summarize data in a clear and simple way. By using tools like graphs, charts, and averages, statistics allow us to see patterns, compare information, and make sense of numbers. Whether it's for research, business, or everyday decisions, statistics turn raw data into useful insights, making complex information easier to understand and act on.
```
Comment (RudraPratik30) [inline docs/docs/widgets/statistics.md:8]: Hey @baibhavKumar1, can you please revert this change? Please update this in both versions.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

File: docs/docs/widgets/statistics.md (hunk 2 above)
Comment (baibhavKumar1) [reply docs/docs/widgets/statistics.md]: Reverted

**Review by RudraPratik30 — APPROVED** [review]
Hey @baibhavKumar1, thanks for contributing!
Approving the changes from my end and forwarding it to the team for final review.
Your PR will be merged by the end of this week. Thanks again!

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Review by adishM98 — APPROVED** [review]
